packages/augur-core/tests/libraries/token/test_standard_token.py

```diff
@@ -162,91 +161,91 @@ def test_transfrom_more_than_total_supply(testStandardTokenFixture):
-    assert standardToken.balanceOf(tester.a1) == 0
-    assert standardToken.balanceOf(tester.a2) == 100
+    assert standardToken.send(testStandardTokenFixture.accounts[2], 100, "", sender=testStandardTokenFixture.accounts[1])
+    assert standardToken.balanceOf(testStandardTokenFixture.accounts[1]) == 0
+    assert standardToken.balanceOf(testStandardTokenFixture.accounts[2]) == 100
 
 def test_operator_send(testStandardTokenFixture):
     standardToken = testStandardTokenFixture.contracts['StandardTokenHelper']
-    assert standardToken.faucet(100, sender = tester.k1)
-    assert standardToken.balanceOf(tester.a1) == 100
-    assert standardToken.balanceOf(tester.a2) == 0
+    assert standardToken.faucet(100, sender = testStandardTokenFixture.accounts[1])
+    assert standardToken.balanceOf(testStandardTokenFixture.accounts[1]) == 100
+    assert standardToken.balanceOf(testStandardTokenFixture.accounts[2]) == 0
 
-    assert not standardToken.isOperatorFor(tester.a0, tester.a1)
+    assert not standardToken.isOperatorFor(testStandardTokenFixture.accounts[0], testStandardTokenFixture.accounts[1])
 
-    with raises(TransactionFailed, message="unauthorized account cannot use operator send"):
-        standardToken.operatorSend(tester.a2, 1)
+    with raises(TransactionFailed):
+        standardToken.operatorSend(testStandardTokenFixture.accounts[2], testStandardTokenFixture.accounts[0], 1, "", "")
 
-    assert standardToken.authorizeOperator(tester.a0, sender=tester.k1)
+    assert standardToken.authorizeOperator(testStandardTokenFixture.accounts[0], sender=testStandardTokenFixture.accounts[1])
 
-    assert standardToken.isOperatorFor(tester.a0, tester.a1)
+    assert standardToken.isOperatorFor(testStandardTokenFixture.accounts[0], testStandardTokenFixture.accounts[1])
 
-    assert standardToken.operatorSend(tester.a1, tester.a2, 1, "", "")
+    assert standardToken.operatorSend(testStandardTokenFixture.accounts[1], testStandardTokenFixture.accounts[2], 1, "", "")
 
-    assert standardToken.balanceOf(tester.a1) == 99
-    assert standardToken.balanceOf(tester.a2) == 1
+    assert standardToken.balanceOf(testStandardTokenFixture.accounts[1]) == 99
+    assert standardToken.balanceOf(testStandardTokenFixture.accounts[2]) == 1
 
-    assert standardToken.revokeOperator(tester.a0, sender=tester.k1)
+    assert standardToken.revokeOperator(testStandardTokenFixture.accounts[0], sender=testStandardTokenFixture.accounts[1])
 
-    assert not standardToken.isOperatorFor(tester.a0, tester.a1)
+    assert not standardToken.isOperatorFor(testStandardTokenFixture.accounts[0], testStandardTokenFixture.accounts[1])
 
-    with raises(TransactionFailed, message="unauthorized account cannot use operator send"):
-        standardToken.operatorSend(tester.a2, 1)
+    with raises(TransactionFailed):
+        standardToken.operatorSend(testStandardTokenFixture.accounts[2], testStandardTokenFixture.accounts[0], 1, "", "")
```

Comment:
I prefer named arguments when the alternative is a long list of mysterious values.

Reply:
This is a contract call (so just a wrapper around an ABI call / transaction). Would be quite a bit of work to add this everywhere.

Reply:
Ah nevermind then.